client/globals.d.ts

```diff
@@ -69,6 +70,9 @@ declare global {
 				declineOnAVSFailure: boolean;
 				declineOnCVCFailure: boolean;
 			};
+			campaigns: {
+				wporgReview2025: boolean;
+			};
```

Comment:
Do we need to document `campaigns` somewhere? Feels like a new concept. Also will all the children of it follow similar rules (e.g. inactive if MIA etc)?

Reply:
Should we use `campaigns.wporgReview2025` to track if merchant has responded or dismissed?

If `campaigns.wporgReview2025` stays boolean, how can we rename it so it's super obvious how it works / what it is?

Reply:
I've added some comments in ed1e318 to clarify this.